src/site/includes/bioParagraph.drupal.liquid

```diff
@@ -0,0 +1,50 @@
+<div class="usa-grid usa-grid-full vads-u-margin-bottom--4">
+  <div class="usa-width-one-fourth">
+    {% assign image = bio.fieldMedia.entity.image %}
+    <img class="circular-profile-image" src="{{ image.derivative.url }}" alt="{{ image.alt }}"
+      title="{{ image.title }}" width="{{ image.derivative.width }}"
+      height="{{ image.derivative.height }}">
+  </div>
+  <div class="usa-width-two-thirds">
+    <h2 class="
+        vads-u-margin-bottom--1
+        vads-u-font-size--md
+        medium-screen:vads-u-font-size--lg">
+      <a href="{{ bio.entityUrl.path }}">{{ bio.fieldNameFirst }}
+        {{ bio.fieldLastName }} {{ bio.fieldSuffix }}</a>
+    </h2>
+    {% if bio.fieldDescription %}
+    <p class="
+        vads-u-font-weight--normal
+        vads-u-margin--0
+        vads-u-margin-bottom--0p5">
+      {{ bio.fieldDescription }}</p>
+    {% endif %}
+    {% if bio.fieldOffice.entity.entityLabel %}
+    <p class="
+        vads-u-font-weight--normal
+        vads-u-margin--0
+        vads-u-margin-bottom--0p5">
+      {{ bio.fieldOffice.entity.entityLabel }}
+    </p>
+    {% endif %}
+    {% if bio.fieldEmailAddress %}
+    <p class="
+        vads-u-font-weight--normal
+        vads-u-margin--0
+        vads-u-margin-bottom--0p5">
+      <b>Email:</b> <a target="blank"
+        href="{{ bio.fieldEmailAddress }}">{{ bio.fieldEmailAddress }}</a>
```

Comment:
should this be a `mailto` href?

Reply:
YES! Thank you for catching that, @ahay-agile6